app/src/components/tasks/TaskList.tsx

```diff
@@ -65,7 +65,7 @@ export default function TaskList(props: TaskListProps) {
           </>
         ) : (
           <>
-            <BackButton onClick={() => {navigate('/', { replace: true })}}/>
```

Comment:
might have happened during the whole name changes proccess, but this should be the back button according to the figma